src/main/app/src/app/taken/taak-view/formio/formio-setup-service.ts

```diff
@@ -235,4 +242,44 @@ export class FormioSetupService {
         ),
     };
   }
+
+  private initializeAvailableDocumentsFieldsetComponent(
+    fieldsetComponent: ExtendedComponentSchema,
+  ): void {
+    if (!this.taak) {
+      return;
+    }
```

Comment:
This is not needed imho. 

We're in private method called by `createFormioForm` that already accepts `taak`.  If the caller did not provide a taak then an exception is ok to be thrown.

Reply:
We can do ` zoekParameters.zaakUUID = this.taak?.zaakUuid;`